data/json/recipes/recipe_companion.json

```diff
@@ -224,6 +224,7 @@
     "time": 56250,
     "autolearn": false,
     "never_learn": true,
+    "flags": ["BLIND_EASY"],
```

Comment:
All files you have touched should be linted (you can use http://dev.narc.ro/cataclysm/format.html for that) - 
Should be `[ "BLIND_EASY" ]` (i.e. single whitspace after `[` and before `]` should be added).

Reply:
Crap, I supposed that where I copied it from it would be ok. Ok.
How is everything not linted yet?